frontend/build.gradle

```diff
@@ -21,7 +21,6 @@ dependencies {
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
```

Comment:
Above this line there is an import for Junit4. Is this the version you settled on? As originally you were going to use Junit 5

Reply:
yh i settled on that one

Reply:
okay you need to remove the stuff around jupiter then in gradle, and make sure that `./gradlew clean test` actually runs your test files.